Scripts/gen_EleBeamResampler.sh

```diff
@@ -1,23 +1,35 @@
 #!/usr/bin/bash
+#
+# create a script for running the EleBeamResampler from an official dataset
+#
+# $1 is the production (ie MDC2020)
+# $2 is the input production version
+# $3 is the output production version
+# $4 is the number of jobs
+if [[ ${4} == "" ]]; then
+  echo "Missing arguments!"
+  return -1
+fi
+
 # create the input list
-dataset=sim.mu2e.EleBeamCat.MDC2020$1.art
-samListLocations --defname="$dataset" > EleBeamCat.txt
+dataset=sim.mu2e.EleBeamCat.$1$2.art
+samweb list-file-locations --schema=root --defname="$dataset"  | cut -f1 > EleBeamCat.txt
 # calucate the max skip from the dataset
 nfiles=`samCountFiles.sh $dataset`
 nevts=`samCountEvents.sh $dataset`
 let nskip=nevts/nfiles
-# write the template fcl
-rm template.fcl
-echo '#include "Production/JobConfig/beam/EleBeamResampler.fcl"' >> template.fcl
-echo physics.filters.beamResampler.mu2e.MaxEventsToSkip: ${nskip} >> template.fcl
+# write the elebeamresampler.fcl
+rm elebeamresampler.fcl
+echo '#include "Production/JobConfig/pileup/EleBeamResampler.fcl"' >> elebeamresampler.fcl
+echo physics.filters.beamResampler.mu2e.MaxEventsToSkip: ${nskip} >> elebeamresampler.fcl
 #
-generate_fcl --dsconf=MDC2020$1 --dsowner=brownd --run-number=1202 --description=EleBeamResampler --events-per-job=200000 --njobs=100 \
-  --embed template.fcl --auxinput=1:physics.filters.beamResampler.fileNames:EleBeamCat.txt
+generate_fcl --dsconf=$1$3 --dsowner=brownd --run-number=1202 --description=EleBeamResampler --events-per-job=400000 --njobs=$4 \
```

Comment:
is there a reason why we doubled the events per job here?

Reply:
I estimate these jobs will take the canonical 2 hours with 400K.   These scripts are intended for use in Validation, if you plan to use them also in the campaign we would have to update that.